core/src/main/resources/org/apache/spark/ui/static/jobs-graph.js

```diff
@@ -0,0 +1,118 @@
+function renderJobsGraphs(data) {
+	/* show visualization toggle */
```

Comment:
Can you use 2 spaces for indents throughout all javascript files, instead of tap characters?